engineyard/ey-cookbooks-dev-v7
# [FBZ-10421] - Migrate Passenger Recipe to v7

**Description of your patch**
Adds Passenger5 by migrating recipe from v6 to v7

**Recommended Release Notes**
Added support for Passenger 5 applications

**Estimated risk**
High

**Components involved**
`ey-passenger5`

**Dependencies**
`ey-lib`

**Description of testing done**
* Create a new environment with v7 as stack
* Indicate "Passenger 5" as the Application for the server stack
* SSH into the instance and execute `monit summary` and check verify if passenger is on the list
* Check crontab via `crontab -l` and verify if `passenger_monitor_<appname>` and `rack_counter_<appname>` is on the list

**QA Instructions**
* Create a new environment with v7 as stack
* Indicate "Passenger 5" as the Application for the server stack
* SSH into the instance and execute `monit summary` and check verify if passenger is on the list
* Check crontab via `crontab -l` and verify if `passenger_monitor_<appname>` and `rack_counter_<appname>` is on the list
